.gitignore

```diff
@@ -8,4 +8,7 @@
 .DS_Store
 
 .1_fetch/tmp/*
-!1_fetch/tmp/state_splits.yml
\ No newline at end of file
+!1_fetch/tmp/state_splits.yml
+
+.3_visualize/out/timeseries_map.html
```

Comment:
it doesn't look like this will ignore the map, since you have a dot "." before `3_visualize`.